site/socket/SocketListener.php

```diff
@@ -0,0 +1,20 @@
+<?php
+    namespace socket;
+
+    use Ratchet\Server\IoServer;
+    use Ratchet\Http\HttpServer;
+    use Ratchet\WebSocket\WsServer;
+    use app\libraries\Socket;
+
+    require_once(__DIR__.'/../vendor/autoload.php');
+
+    $server = IoServer::factory(
+        new HttpServer(
+                new WsServer(
+                    new Socket()
+                )
+        ),
+        8080
+    );
```

Comment:
Can we limit this to only listening on `127.0.0.1` (after doing the aforementioned apache stuff) and not have this port be public, or is this something we'd have to do via iptables?